brave/brave-core
# Import cookies from Firefox

Resolves brave/brave-browser#343.

Confirmed builds and successfully passes manual and automated tests on Windows, macOS, and Linux.

## Submitter Checklist:


## Test Plan:

1. Set some cookies in a Firefox profile
2. `yarn start` with a fresh user data directory
3. Open the main menu (e.g. **Brave** on macOS) and choose **Import Bookmarks and Settings...**
4. Choose the Firefox profile from the dropdown menu of browser profiles.
5. Make sure the **Cookies** checkbox is activated.
6. Click **Import**.
7. Go to `chrome://settings/siteData?search=cookies` and confirm that the expected cookies were imported.

## Reviewer Checklist:
